[2025-03-10T00:11:21.363+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_maestro.load_data manual__2025-03-09T23:59:49.142708+00:00 [queued]>
[2025-03-10T00:11:21.388+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_maestro.load_data manual__2025-03-09T23:59:49.142708+00:00 [queued]>
[2025-03-10T00:11:21.388+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-10T00:11:21.424+0000] {taskinstance.py:1350} INFO - Executing <Task(TriggerDagRunOperator): load_data> on 2025-03-09 23:59:49.142708+00:00
[2025-03-10T00:11:21.447+0000] {standard_task_runner.py:57} INFO - Started process 837 to run task
[2025-03-10T00:11:21.466+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_maestro', 'load_data', 'manual__2025-03-09T23:59:49.142708+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/orchestrator.py', '--cfg-path', '/tmp/tmp450pt5lf']
[2025-03-10T00:11:21.467+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask load_data
[2025-03-10T00:11:21.627+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_maestro.load_data manual__2025-03-09T23:59:49.142708+00:00 [running]> on host d7c1c9924032
[2025-03-10T00:11:21.893+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_maestro' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T23:59:49.142708+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T23:59:49.142708+00:00'
[2025-03-10T00:11:22.118+0000] {trigger_dagrun.py:201} INFO - Waiting for load_data_mysql on 2025-03-10T00:11:21.894343+00:00 to become allowed state ['success'] ...
[2025-03-10T00:12:22.198+0000] {trigger_dagrun.py:210} INFO - load_data_mysql finished with allowed state success
[2025-03-10T00:12:22.214+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=dag_maestro, task_id=load_data, execution_date=20250309T235949, start_date=20250310T001121, end_date=20250310T001222
[2025-03-10T00:12:22.271+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-10T00:12:22.322+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
